yarn.lock

```diff
@@ -860,6 +1177,98 @@
     "@ethersproject/properties" "^5.4.0"
     "@ethersproject/strings" "^5.4.0"
 
+"@keep-network/keep-core@1.8.0-ropsten.0":
+  version "1.8.0-ropsten.0"
+  resolved "https://registry.yarnpkg.com/@keep-network/keep-core/-/keep-core-1.8.0-ropsten.0.tgz#0ebc5f4b7c715d54d669e1ace1b49c62ef035527"
+  integrity sha512-6AgwR4zVPX6Lsbhxn+Q/xFA5MtQA4XKVxR4iHztkNeq2BKr+yNXkmKm1cOM0uSXbu38CSsubYXzkjguy67kFcQ==
+  dependencies:
+    "@openzeppelin/upgrades" "^2.7.2"
+    openzeppelin-solidity "2.4.0"
+
+"@keep-network/keep-core@^1.8.0-dev":
+  version "1.8.0-ropsten.2"
+  resolved "https://registry.yarnpkg.com/@keep-network/keep-core/-/keep-core-1.8.0-ropsten.2.tgz#4b0d4810fa4ae41970cc40c434ab5cc460eca928"
+  integrity sha512-OWuafRUQicjHZImzF0irw0iE+lmxF4wTKyI4kNuht/2O1v/KqRSMM57onUT6rOBRiKVXAX/EjMW0kiJwT2Zdgg==
+  dependencies:
+    "@openzeppelin/upgrades" "^2.7.2"
+    openzeppelin-solidity "2.4.0"
```

Comment:
Should we have both `keep-core` dependencies here?

Reply:
Yes, the dependencies were auto-resolved by yarn, and both are required, please see:
```
➜  coverage-pools git:(deployment-external-contracts) ✗ yarn why @keep-network/keep-core                    
yarn why v1.17.3
[1/4] 🤔  Why do we have the module "@keep-network/keep-core"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@keep-network/keep-core@1.8.0-dev.1"
info Has been hoisted to "@keep-network/keep-core"
info This module exists because it's specified in "dependencies".
info Disk size without dependencies: "67.52MB"
info Disk size with unique dependencies: "103.96MB"
info Disk size with transitive dependencies: "266.96MB"
info Number of shared dependencies: 201
=> Found "@keep-network/keep-ecdsa#@keep-network/keep-core@1.8.0-rc.6"
info This module exists because "@keep-network#tbtc#@keep-network#keep-ecdsa" depends on it.
info Disk size without dependencies: "67.55MB"
info Disk size with unique dependencies: "103.98MB"
info Disk size with transitive dependencies: "266.99MB"
info Number of shared dependencies: 201
✨  Done in 2.92s.
```